en-us.yml

```diff
@@ -1,7 +1,50 @@
 # Loritta's English Translation
 # Authors:
 # MrPowerGamerBR (https://mrpowergamerbr.com/)
+#ItsGabii
+discord:
+  permissions:
+    kick-members: "Kick members"
+    ban-members: "Ban members"
+    administrator: "Administrador"
+    manage-channel: "Manage channel"
+    manage-server: "Manage server"
+    add-reactions: "Add reacions"
+    message-history: "See message history"
+    view-audit-logs: "See Audit Logs"
+    message-read: "Read messages"
+    message-write: "Send messages"
+    message-manage: "Manage messages"
+    message-ext-emoji: "Use external emojis"
+    message-embed-links: "insert links"
+    manage-roles: "Manage roles"
+    manage-permissions: "Manage permissions"
+    manage-webhooks: "Manage Webhooks"
+    manage-emotes: "Manage emojis"
+    create-instant-invite: "Create an instant invite"
 commands:
+please-wait-cooldown: "Calm down! you need to wait more **{0}** before execute another command! {1}"
+  error-while-executing-command: "Something gone extremaly wrong in executing this command... 😭 Sorry for the inconvenience. 😢"

```

Comment:
Something went extremely wrong when executing this command

Reply:
while executing...?